src/main/java/org/broadinstitute/hellbender/tools/dataflow/MarkDuplicatesDataflow.java

```diff
@@ -344,9 +622,15 @@ private static int score(final GATKRead record) {
     /**
      * Struct-like class to store information about the paired reads for mark duplicates.
      */
-    private static final class PairedEnds {
+    private static final class PairedEnds implements OpticalDuplicateFinder.PhysicalLocation {
```

Comment:
Since the values added by `OpticalDuplicateFinder` seem to only be used in OpticalDuplicate finder it would be nice if they didn't have to be exposed here.